mscore/musescore.cpp

```diff
@@ -6880,6 +6880,9 @@ int main(int argc, char* av[])
 #endif
 
       QApplication::setDesktopSettingsAware(true);
+#ifdef Q_OS_LINUX
+      QGuiApplication::setDesktopFileName("mscore");
+#endif
```

Comment:
Looks good to me!

For those not in the know, it is referring to [this file](https://github.com/musescore/MuseScore/blob/bafeb45af86e773d5860da9c49bca732e043a13b/build/Linux%2BBSD/mscore.desktop.in). Here is an [example](https://github.com/flatpak/qt-flatpak-demo/blob/0dae127ad4698e65a8c1730614f6e07d5b8f044c/main.cpp#L36) of it being used in another application.

Qt Documentation is here: https://doc.qt.io/qt-5/qguiapplication.html#desktopFileName-prop